Neos.Cache/Resources/Private/DDL_pgsql.sql

```diff
@@ -0,0 +1,22 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS "cache" (
+  "identifier" VARCHAR(250) NOT NULL,
+  "cache" VARCHAR(250) NOT NULL,
+  "context" VARCHAR(150) NOT NULL,
+  "created" INTEGER NOT NULL,
```

Comment:
I saw a recommendation to use `"created" INTEGER CHECK(created >= 0) NOT NULL,` to replace the `UNSIGNED` config.

Reply:
Sorry for the (very) late reaction...
what does this do and is this syntax compatible with all supported platforms?

Reply:
nevermind – this PR is totally outdated